packages/storybook/stories/ParticipantItem/snippets/CustomIcon.snippet.tsx

```diff
@@ -14,13 +14,16 @@ export const CustomIconExample: () => JSX.Element = () => {
 
   const containerStyle = mergeStyles({ width: '15rem' });
 
+  const participantItemStyle: ParticipantItemStyles = { me: { fontFamily: 'Segoe UI' } };
```

Comment:
This shouldn't be required -- do you know why we need to set this? This should be the default font.

Reply:
Actually, can you check where the `(you)` text is in the ParticipantItem, from a quick glance it might need wrapped in a `<Text>` block (Text imported from fluentui)

Reply:
This is where the (you) text is 
<img width="496" alt="Screen Shot 2022-04-29 at 4 31 33 PM" src="https://user-images.githubusercontent.com/96077406/166080671-3d4aa294-fb89-4577-8173-5ae6872f6b9e.png">

Reply:
displayed names are generated using fluent ui <persona>, which explains why they are different font

Reply:
Tried wrapping it in a <text> block, the color changed to the same color as participant name. We still need to apply extra styling to change the text color.

Reply:
Tried wrapping it in a Text block, it works! Updated the PR :)